diff -uNr nagios-plugin-check_apc-0.0.2.orig/check_apc nagios-plugin-check_apc-0.0.2/check_apc
--- nagios-plugin-check_apc-0.0.2.orig/check_apc	2006-02-21 13:00:57.000000000 +0100
+++ nagios-plugin-check_apc-0.0.2/check_apc	2006-02-21 12:47:19.000000000 +0100
@@ -21,7 +21,7 @@
 ## Foundation, Inc., 59 Temple Place, Suite 330, Boston, MA  02111-1307  USA
 
 
-APCA="/sbin/apcaccess"
+APCA="/usr/sbin/apcaccess"
 APCA_ARGS="status"
 
 #APCA="apcsnmp.pl"
